### DISCUSSION TITLE
Access Wazuh Dashboard

### DISCUSSION THREAD

**Discussion by OlexTratisky** [Security-Onion-Solutions/securityonion#9138] [General] [answered]
Hi,

I would like to know if there is a way to access the Wazuh manager dashboard https:// interface to configure Wazuh from the browser

Thanks

**Comment by dougburks** [comment, marked as answer]
We include dashboards for slicing and dicing Wazuh data, but we do not include the Wazuh plugin for Kibana for configuration.

**Reply by gitmffonseca** [reply]
Hi @dougburks !
Can you explain why Wazuh plugin not included?
Will it be in the near future?
Thank you

**Reply by UMHB-InfoSec** [reply]
I can't say for sure, but I did get a notification several days ago that request 237 was closed as completed.

https://github.com/Security-Onion-Solutions/securityonion/issues/237

So, maybe its planned for 2.4?

**Reply by dougburks** [reply]
We have no plans to include the Wazuh plugin for Kibana.

**Reply by gitmffonseca** [reply]
A little bit strange considering this: https://github.com/Security-Onion-Solutions/securityonion/issues/237

Any alternative in SO2 to visualize CIS-Benchmarks, Vulnerabilities, Security Events, Policies Enforcing for endpoints in GUI?

Thanks

**Reply by dougburks** [reply]
> A little bit strange considering this: #237

https://github.com/Security-Onion-Solutions/securityonion/issues/237 is from January 2020 and we were considering the Wazuh plugin for Kibana at that time. We are no longer considering it and the issue has been closed.

> Any alternative in SO2 to visualize CIS-Benchmarks, Vulnerabilities, Security Events, Policies Enforcing for endpoints in GUI?

You can do some of this today using [FleetDM](https://docs.securityonion.net/en/2.3/fleet.html). In the upcoming Security Onion 2.4, you will be able to do some of this using [Elastic Fleet](https://www.elastic.co/guide/en/fleet/current/fleet-overview.html).

**Reply by gitmffonseca** [reply]
Thank you @dougburks  for the feedback!
I confess feel a little bit disappointed after more than 2 years for the Wazuh feature to be dropped right now!

From your answer, only some of these features can and will be fulfilled…!

Thanks anyway!
